src/test/java/spoon/test/targeted/TargetedExpressionTest.java

```diff
@@ -265,7 +252,7 @@ public void testStaticTargetsOfFieldAccessNoClasspath() throws Exception {
 		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(exepectedThisAccess).result("foo"), elements.get(2));
 		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(exepectedThisAccess).result("Foo.this.i"), elements.get(3));
 		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(exepectedThisAccess).result("foo"), elements.get(4));
-		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(expectedTypeAccess).result("Foo.staticField"), elements.get(5));
+		assertEqualsFieldAccess(new ExpectedTargetedExpression().declaringType(expectedFoo).target(expectedTypeAccess.toString()).result("Foo.staticField"), elements.get(5));
```

Comment:
The target is a `CtCodeSnippetExpression`. For now, it isn't necessary but it'll be with the equals refactoring.